docs/UserGuide.adoc

```diff
@@ -151,6 +151,10 @@ Views all details of the 2nd person in the address book.
 `viewall 1` +
 Views all details of the 1st person in the results of the `find` command.
 
+== Clearing all entries : `sort`
```

Comment:
Are you sure you updated the user guide correctly? I would not want a sort command that clears all my entries 😲

Also, it would be good to add a sort description of what the command does.